components/gitpod-protocol/src/util/gitpod-cookie.ts

```diff
@@ -12,6 +15,10 @@ import * as cookie from "cookie";
 export const NAME = "gitpod-user";
 export const VALUE = "true";
 
+export const NECESSARY_COOKIES = "gp-necessary";
```

Comment:
The (single) cookie name needs to contain the domain name. E.g. for `gitpod.io`, it could be prefixed with `gitpod-io-consent`.

Reply:
cc @nisarhassan12 so we will align